libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/CountryBorderCrossingInfo.kt

```diff
@@ -0,0 +1,68 @@
+package com.mapbox.navigation.base.trip.model.alert
+
+/**
+ * Administrative information.
+ *
+ * @param from origin administrative info when crossing the country border
+ * @param to destination administrative info when crossing the country border
+ */
+class CountryBorderCrossingInfo private constructor(
```

Comment:
Should we add a `CountryBorderCrossingInfo` `BuilderTest`? Noticed that we didn't add these for the builders in `trip.model.eh`, was that on purpose?

Reply:
added